src/components/AuthEvents/EditEvent.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import ReactDOM from 'react-dom';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import DatePicker from 'material-ui/DatePicker';
+
+//components
+import Navigation from '../Navigation/Navigation';
+import Footer from '../Footer/Footer';
+import Category from '../category/GetCategories';
+
+import { getEventDetail } from '../../actions/eventActions';
+import { editEventAction } from '../../actions/authEvents.actions';
+
+
+const initialState = {
+	title: '',
+	location: '',
+	time: '',
+	date: '',
+	description: '',
+	image_url:'',
+	event_category: null
+};
+
+
+class EditEvent extends Component{
+
+    constructor(props){
+		super(props);
+        this.state = {
+			eventData: initialState,
+		};
+		// console.log('Constructor',editEvent);
+		this.handleInputChange = this.handleInputChange.bind(this);
+		this.handleCategoryChange = this.handleCategoryChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+
+	componentWillReceiveProps(nextProps){
+		this.setState({eventData: nextProps.editEvent.event})
+	}
+
+    componentWillMount = () => {
+		let eventId = this.props.match.params.id
```

Comment:
Indentation Issue here
Also, check you linter rules as some places you are using semis others, you are not